### ISSUE TITLE
Don't send SketchModeEnable or Disable when KCL builds paths

### ISSUE THREAD

**Issue by adamchalmers** [KittyCAD/modeling-app#1973]
If you run the test case [riddle_small.kcl](https://github.com/KittyCAD/modeling-app/blob/main/src/wasm-lib/tests/executor/inputs/riddle_small.kcl) you'll see that it sends modeling commands `SketchModeEnable` and `SketchModeDisable`. I don't think these are actually necessary. They should be removed, they're just adding unnecessary IO.

**Comment by jessfraz** [comment]
i actaully think we needed these for some god forsaken reason, but i cant remember anyways worth trying ill try and remember i think theres a code comment somewhere

**Comment by jessfraz** [comment]
yeah something broke on sketch on face where this has to be done after extrude, aka extrude for sketch on face needs to be in sketch mode https://github.com/KittyCAD/modeling-app/blob/main/src/wasm-lib/kcl/src/std/extrude.rs#L60 but yeah would be nice if the engine didnt force that

**Comment by jessfraz** [comment]
we cleaned a lot of this up with @benjamaan476 help, only 2 places remain:

## Problem 1: starting a sketch 

in the limit we will want something like to replace `EnableSketchMode`, `StartPath` into one, since thats all we are doing, https://github.com/KittyCAD/modeling-app/blob/44be072d04cd75e5e817af64f32a784f0d4377a2/rust/kcl-lib/src/std/sketch.rs#L1240:
```
pub struct StartSketch {
   /// The plane or face id to sketch on.
    pub entity_id: [Uuid](https://docs.rs/uuid/1.15.1/x86_64-unknown-linux-gnu/uuid/struct.Uuid.html),
    pub planar_normal: [Option](https://doc.rust-lang.org/nightly/core/option/enum.Option.html)<[Point3d](https://docs.rs/kittycad-modeling-cmds/latest/kittycad_modeling_cmds/shared/struct.Point3d.html)<[f64](https://doc.rust-lang.org/nightly/std/primitive.f64.html)>>,
}
```

where the cmd_id sent with the cmd becomes the path id for the sketch / sketch id.

The after we can call MovePathPen and use the id for that segment as teh cmd_id like we are doing already. BUt this will need to work outside the traditional sketch mode.



## Problem 2 extruding

we still have to enable sketch mode so teh extrude is in the right direction, if instead by default it just went the right way , that would be awesome
